flocker/control/test/test_httpapi.py

```diff
@@ -242,6 +242,79 @@ def test_container_name_collision_different_node(self):
         """
         return self._container_name_collision_test(self.NODE_A, self.NODE_B)
 
+    def test_create_container_with_environment(self):
+        """
+        A valid API request to create a container including environment
+        variables results in an updated configuration.
+        """
+        saving = self.persistence_service.save(Deployment(
+            nodes={
+                Node(
+                    hostname=self.NODE_A,
+                    applications=[]
+                ),
+                Node(hostname=self.NODE_B),
+            }
+        ))
```

Comment:
What's this bit for? To wipe out the configuration before performing the test? 
Add an explanatory comment.
Move this to `setUp` ?